### PR TITLE
Fix a crash when multi-turreted units leave a transport.

### LINKED ISSUE #11171: Actor x has multiple traits of type `OpenRA.Mods.Common.Traits.Turreted`
Hello, I figured this isn't supposed to happen, but this error is thrown whenever a unit with more than 1 turret leaves a unit that can carry it in it's cargo (IE water transport or paradrop). It is relatively easy to reproduce, make a tank with 2 turrets, Turreted@primary, @secondary, etc. then insert it into a paradrop power or into a water transport, any attempt to empty the cargo with this unit inside it will cause the game to throw the error.

### PR DESCRIPTION
**Pull request by pchote**
Fixes #11171.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Cargo.cs [hunk 1]
```diff
@@ -266,9 +266,8 @@ void SetPassengerFacing(Actor passenger)
 			if (passengerFacing != null)
 				passengerFacing.Facing = facing.Value.Facing + Info.PassengerFacing;
 
-			var passengerTurreted = passenger.TraitOrDefault<Turreted>();
-			if (passengerTurreted != null)
-				passengerTurreted.TurretFacing = facing.Value.Facing + Info.PassengerFacing;
+			foreach (var t in passenger.TraitsImplementing<Turreted>())
+				t.TurretFacing = facing.Value.Facing + Info.PassengerFacing;
```
Comment (Phrohdoh) [inline OpenRA.Mods.Common/Traits/Cargo.cs]: Check for disabled traits, too?

Comment (pchote) [inline OpenRA.Mods.Common/Traits/Cargo.cs]: Turreted doesn't support disabling.

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

**Comment by reaperrr** [comment]
:+1: 
Test commit can be removed now.

**Comment by pchote** [comment]
Removed test commit.

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/4ffd8e4b642a9bf74393273351aeea46c3cafafb...442d3fedddc093fcbb417ff9d109e0f636dc8632)

**Comment by obrakmann** [comment]
Moved changelog entry to [Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/5c90b72be7cd258e3e25b622ba0503bab884acc0)

**Pull request merged**